src/stactools/core/utils/antimeridian.py

```diff
@@ -10,14 +11,17 @@
 
 
 class Strategy(Enum):
-    """Strategy for handling antimeridian-crossing polygons.
+    """Strategy for handling antimeridian-crossing polygons."""
 
-    - SPLIT: split the polygon at the antimeridian
-    - NORMALIZE: keep the polygon as one, but extend it to > 180 or < -180.
+    SPLIT = auto()
+    """Split the polygon into two or more polygons, such that none of them cross
+    the antimeridian.
     """
```

Comment:
I've not seen this, where triple quotes are used underneath a definition. I'm guessing there is for some sphinx magic?

Reply:
Nope, just total habit for putting the main subject of the docs on the same line as the starting `"""` but wanting a newline before the last `"""`. It's a bastardization of https://peps.python.org/pep-0257/#multi-line-docstrings, which says that the first line should never wrap, but sometimes I wrap them for the sake of not spending a lot of brain cells on shortening the description. I'll shorten this one.